src/Storage/Doctrine/Connection.php

```diff
@@ -181,10 +175,10 @@ private function addTableToSchema(Schema $schema): void
         $table->addColumn('id', Types::INTEGER)->setNotnull(true)->setAutoincrement(true);
         $table->addColumn('message_uid', Types::GUID)->setNotnull(true);
         $table->addColumn('class', Types::STRING)->setLength(255)->setNotnull(true);
-        $table->addColumn('dispatched_at', Types::DATETIME_IMMUTABLE)->setNotnull(true);
-        $table->addColumn('received_at', Types::DATETIME_IMMUTABLE)->setNotnull(false);
-        $table->addColumn('handled_at', Types::DATETIME_IMMUTABLE)->setNotnull(false);
-        $table->addColumn('failed_at', Types::DATETIME_IMMUTABLE)->setNotnull(false);
+        $table->addColumn('dispatched_at', Types::FLOAT)->setNotnull(true);
+        $table->addColumn('waiting_time', Types::FLOAT)->setNotnull(false);
+        $table->addColumn('handling_time', Types::FLOAT)->setNotnull(false);
+        $table->addColumn('failing_time', Types::FLOAT)->setNotnull(false);
```

Comment:
Just an FYI for us, this will require a migration - iirc, `make:migration` will see this changes, but it's been awhile so I'm not 100% sure.

Reply:
looks like you can backfill these new columns, as they are calculated from the existing ones?
the existing ones can then be dropped